### ISSUE TITLE
[neo-cli 2.9.3] Stack trace when system is nearly out of memory

### ISSUE THREAD

**Issue by ixje** [neo-project/neo-cli#272]
```
neo> [ERROR][2018-12-14 15:17:10][Thread 0036][[akka://NeoSystem/system/IO-TCP/$Yk#420013725]] Could not establish connection to [47.75.183.166:10333].
Cause: System.Net.Sockets.SocketException (0x80004005): An operation on a socket could not be performed because the system lacked sufficient buffer space or because a queue was full
   at System.Net.Sockets.Socket.UpdateStatusAfterSocketErrorAndThrowException(SocketError error, String callerName)
   at System.Net.Sockets.Socket.DoBind(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.Sockets.Socket.InternalBind(EndPoint localEP)
   at System.Net.Sockets.Socket.ConnectAsync(SocketAsyncEventArgs e)
   at Akka.IO.TcpOutgoingConnection.<>c__DisplayClass11_0.<Register>b__0()
   at Akka.IO.TcpOutgoingConnection.ReportConnectFailure(Action thunk)
```
My system was running out of physical memory when importing the chain from offline files and the cli continuously throws this trace. It would be a better user experience if it was gracefully handled and perhaps a normal "Nearly out of memory" error was shown.

**Comment by erikzhang** [comment]
When importing blocks, there is no way for the system to process messages sent by other nodes. These messages will accumulate in the queue, resulting in a large amount of memory.

**Comment by shargon** [comment]
Maybe we can `sleep` when the queue is too high

**Comment by jsolman** [comment]
When importing blocks, I find that it is easier to just not call `system.StartNode` until after the import is complete. One way to achieve this is to add a command to start the node and then issue it after show state indicates that importing is complete. A command line argument could be used to indicate not to start the node initially.

**Comment by erikzhang** [comment]
https://github.com/neo-project/neo/pull/523

**Comment by atomlab** [comment]
I have same problem. I had run import blockchain from chain.acc. neo-cli used 13G of physical memory.

**Comment by erikzhang** [comment]
This issue has been fixed in neo-project/neo#523.
